burn-dataset/src/source/huggingface/downloader.rs

```diff
@@ -205,24 +244,32 @@ fn importer_script_path(base_dir: &Path) -> PathBuf {
 }
 
 fn install_python_deps(base_dir: &Path) -> Result<PathBuf, ImporterError> {
+    let python_name = get_python_name()?;
```

Comment:
I don't think we need to call `get_python_name` if the venv is already initialized. Can we call this function only when creating a new venv?

Reply:
What you said is correct, and it has been changed now.